js/entryView.js

```diff
@@ -19,6 +9,24 @@ entryView.handleMainNav = function() {
   });
 };
 
+entryView.create = function() {
+  var entry;
+  $('#entries').empty();
+  entry = new Entry({
+    sitename: $('#entry-sitename').val(),
+    siteUrl: $('#entry-siteurl').val(),
+    description: $('#entry-description').val()
+  });
+  $('#entries').append(entry.toHtml());
+
+};
+
+entryView.initIndexPage = function() {
+  Entry.all.forEach(function(a) {
+    $('#entries').append(a.toHtml());
+  });
+};
```

Comment:
Try to avoid using ambiguous variable names (single characters). As someone who is unfamiliar with your code, I have no idea what an "a" represents without needing to trace through the code.